2025/08/27 13:05:43 [emerg] 1#1: unknown directive "lua_package_path" in /etc/nginx/nginx.conf:17
2025/08/27 13:10:38 [emerg] 1#1: unknown directive "lua_package_path" in /etc/nginx/nginx.conf:17
2025/08/27 13:21:39 [emerg] 1#1: unknown directive "lua_package_path" in /etc/nginx/nginx.conf:17
2025/08/27 13:22:19 [emerg] 1#1: host not found in upstream "financial-service:8002" in /etc/nginx/nginx.conf:93
2025/08/27 13:23:24 [emerg] 1#1: host not found in upstream "financial-service:8002" in /etc/nginx/nginx.conf:93
2025/08/27 13:23:53 [emerg] 1#1: "set" directive is not allowed here in /etc/nginx/conf.d/locations.conf:5
2025/08/27 13:24:20 [emerg] 1#1: "location" directive is not allowed here in /etc/nginx/conf.d/locations.conf:8
2025/08/27 13:24:43 [emerg] 1#1: "location" directive is not allowed here in /etc/nginx/conf.d/locations.conf:5
2025/08/27 13:25:32 [emerg] 1#1: "location" directive is not allowed here in /etc/nginx/conf.d/locations.conf:5
2025/08/27 13:26:18 [warn] 1#1: the "listen ... http2" directive is deprecated, use the "http2" directive instead in /etc/nginx/conf.d/wl-school.conf:76
2025/08/27 13:26:18 [emerg] 1#1: cannot load certificate "/etc/nginx/ssl/wl-school.crt": BIO_new_file() failed (SSL: error:80000002:system library::No such file or directory:calling fopen(/etc/nginx/ssl/wl-school.crt, r) error:10000080:BIO routines::no such file)
2025/08/27 13:27:53 [error] 23#23: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /?ide_webview_request_time=1756301273084 HTTP/1.1", upstream: "http://172.20.0.7:3000/?ide_webview_request_time=1756301273084", host: "localhost:8000"
2025/08/27 13:27:53 [error] 23#23: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /@vite/client HTTP/1.1", upstream: "http://172.20.0.7:3000/@vite/client", host: "localhost:8000", referrer: "http://localhost:8000/?ide_webview_request_time=1756301273084"
2025/08/27 13:28:22 [error] 25#25: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.20.0.7:3000/", host: "localhost:8000"
2025/08/27 13:28:22 [error] 25#25: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.20.0.7:3000/favicon.ico", host: "localhost:8000", referrer: "http://localhost:8000/"
2025/08/27 13:29:18 [error] 25#25: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.20.0.7:3000/", host: "localhost:8000"
2025/08/27 13:29:18 [error] 25#25: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", upstream: "http://172.20.0.7:3000/.well-known/appspecific/com.chrome.devtools.json", host: "localhost:8000"
2025/08/27 13:29:18 [error] 25#25: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.20.0.7:3000/favicon.ico", host: "localhost:8000", referrer: "http://localhost:8000/"
2025/08/27 13:30:55 [error] 20#20: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /?ide_webview_request_time=1756301455681 HTTP/1.1", upstream: "http://172.20.0.7:3000/?ide_webview_request_time=1756301455681", host: "localhost:8000"
2025/08/27 13:30:55 [error] 20#20: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /@vite/client HTTP/1.1", upstream: "http://172.20.0.7:3000/@vite/client", host: "localhost:8000", referrer: "http://localhost:8000/?ide_webview_request_time=1756301455681"
2025/08/27 13:32:00 [error] 20#20: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /?ide_webview_request_time=1756301455681 HTTP/1.1", upstream: "http://172.20.0.7:3000/?ide_webview_request_time=1756301455681", host: "localhost:8000"
2025/08/27 13:32:00 [error] 20#20: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /@vite/client HTTP/1.1", upstream: "http://172.20.0.7:3000/@vite/client", host: "localhost:8000", referrer: "http://localhost:8000/?ide_webview_request_time=1756301455681"
2025/08/27 13:32:05 [error] 20#20: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.20.0.7:3000/", host: "localhost:8000"
2025/08/27 13:32:06 [error] 20#20: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", upstream: "http://172.20.0.7:3000/.well-known/appspecific/com.chrome.devtools.json", host: "localhost:8000"
2025/08/27 13:32:06 [error] 20#20: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.20.0.7:3000/favicon.ico", host: "localhost:8000", referrer: "http://localhost:8000/"
2025/08/27 13:32:07 [error] 20#20: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.20.0.7:3000/", host: "localhost:8000"
2025/08/27 13:32:07 [error] 20#20: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", upstream: "http://172.20.0.7:3000/.well-known/appspecific/com.chrome.devtools.json", host: "localhost:8000"
2025/08/27 13:32:07 [error] 20#20: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.20.0.7:3000/favicon.ico", host: "localhost:8000", referrer: "http://localhost:8000/"
2025/08/27 13:33:41 [error] 20#20: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /?ide_webview_request_time=1756301455681 HTTP/1.1", upstream: "http://172.20.0.7:3000/?ide_webview_request_time=1756301455681", host: "localhost:8000"
2025/08/27 13:33:41 [error] 20#20: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /@vite/client HTTP/1.1", upstream: "http://172.20.0.7:3000/@vite/client", host: "localhost:8000", referrer: "http://localhost:8000/?ide_webview_request_time=1756301455681"
2025/08/27 13:33:43 [error] 20#20: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /?ide_webview_request_time=1756301623625 HTTP/1.1", upstream: "http://172.20.0.7:3000/?ide_webview_request_time=1756301623625", host: "localhost:8000"
2025/08/27 13:33:43 [error] 20#20: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /@vite/client HTTP/1.1", upstream: "http://172.20.0.7:3000/@vite/client", host: "localhost:8000", referrer: "http://localhost:8000/?ide_webview_request_time=1756301623625"
2025/08/27 13:33:44 [error] 20#20: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /?ide_webview_request_time=1756301623625 HTTP/1.1", upstream: "http://172.20.0.7:3000/?ide_webview_request_time=1756301623625", host: "localhost:8000"
2025/08/27 13:33:44 [error] 20#20: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /@vite/client HTTP/1.1", upstream: "http://172.20.0.7:3000/@vite/client", host: "localhost:8000", referrer: "http://localhost:8000/?ide_webview_request_time=1756301623625"
2025/08/27 13:33:57 [error] 20#20: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.20.0.7:3000/", host: "localhost:8000"
2025/08/27 13:33:57 [error] 20#20: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", upstream: "http://172.20.0.7:3000/.well-known/appspecific/com.chrome.devtools.json", host: "localhost:8000"
2025/08/27 13:33:57 [error] 20#20: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.20.0.7:3000/favicon.ico", host: "localhost:8000", referrer: "http://localhost:8000/"
2025/08/27 13:33:58 [error] 20#20: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.20.0.7:3000/", host: "localhost:8000"
2025/08/27 13:33:58 [error] 20#20: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", upstream: "http://172.20.0.7:3000/.well-known/appspecific/com.chrome.devtools.json", host: "localhost:8000"
2025/08/27 13:33:58 [error] 20#20: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.20.0.7:3000/favicon.ico", host: "localhost:8000", referrer: "http://localhost:8000/"
2025/08/27 13:37:25 [error] 22#22: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.20.0.7:3000/", host: "localhost:8000"
2025/08/27 13:37:26 [error] 22#22: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", upstream: "http://172.20.0.7:3000/.well-known/appspecific/com.chrome.devtools.json", host: "localhost:8000"
2025/08/27 13:37:26 [error] 22#22: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.20.0.7:3000/favicon.ico", host: "localhost:8000", referrer: "http://localhost:8000/"
2025/08/27 15:04:10 [warn] 20#20: *1 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/1/00/0000000001 while reading upstream, client: 192.168.65.1, server: localhost, request: "GET /static/js/main.5d5b3854.js.map HTTP/1.1", upstream: "http://172.20.0.7:80/static/js/main.5d5b3854.js.map", host: "localhost:8000"
2025/08/27 16:07:13 [warn] 22#22: *2 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/1/00/0000000001 while reading upstream, client: 192.168.65.1, server: localhost, request: "GET /static/js/main.d87faa2e.js.map HTTP/1.1", upstream: "http://172.20.0.7:80/static/js/main.d87faa2e.js.map", host: "localhost:8000"
2025/08/27 16:26:38 [error] 23#23: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /?ide_webview_request_time=1756311997731 HTTP/1.1", upstream: "http://172.20.0.7:80/?ide_webview_request_time=1756311997731", host: "localhost:8000"
2025/08/27 16:26:40 [error] 23#23: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /@vite/client HTTP/1.1", upstream: "http://172.20.0.7:80/@vite/client", host: "localhost:8000", referrer: "http://localhost:8000/?ide_webview_request_time=1756311997731"
2025/08/27 16:26:52 [error] 23#23: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /login HTTP/1.1", upstream: "http://172.20.0.7:80/login", host: "localhost:8000"
2025/08/27 16:26:55 [error] 23#23: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", upstream: "http://172.20.0.7:80/.well-known/appspecific/com.chrome.devtools.json", host: "localhost:8000"
2025/08/27 16:32:46 [error] 23#23: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /login HTTP/1.1", upstream: "http://172.20.0.7:80/login", host: "localhost:8000"
2025/08/27 16:32:46 [error] 23#23: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", upstream: "http://172.20.0.7:80/.well-known/appspecific/com.chrome.devtools.json", host: "localhost:8000"
2025/08/27 16:32:48 [error] 23#23: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /login HTTP/1.1", upstream: "http://172.20.0.7:80/login", host: "localhost:8000"
2025/08/27 16:32:48 [error] 23#23: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1", upstream: "http://172.20.0.7:80/.well-known/appspecific/com.chrome.devtools.json", host: "localhost:8000"
2025/08/27 16:40:21 [error] 21#21: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /?ide_webview_request_time=1756312821861 HTTP/1.1", upstream: "http://172.20.0.7:80/?ide_webview_request_time=1756312821861", host: "localhost:8000"
2025/08/27 16:40:22 [error] 21#21: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /@vite/client HTTP/1.1", upstream: "http://172.20.0.7:80/@vite/client", host: "localhost:8000", referrer: "http://localhost:8000/?ide_webview_request_time=1756312821861"
2025/08/27 17:35:29 [error] 21#21: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "OPTIONS /api/auth/login HTTP/1.1", upstream: "http://172.20.0.5:8000/api/auth/login", host: "localhost:8000", referrer: "http://localhost:3000/"
2025/08/27 17:35:56 [error] 21#21: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "OPTIONS /api/auth/login HTTP/1.1", upstream: "http://172.20.0.5:8000/api/auth/login", host: "localhost:8000", referrer: "http://localhost:3000/"
2025/08/27 17:37:22 [error] 21#21: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "OPTIONS /api/auth/login HTTP/1.1", upstream: "http://172.20.0.5:8000/api/auth/login", host: "localhost:8000", referrer: "http://localhost:3000/"
2025/08/27 17:38:59 [error] 21#21: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "POST /api/auth/login HTTP/1.1", upstream: "http://172.20.0.5:8000/api/auth/login", host: "localhost:8000"
2025/08/27 17:40:35 [error] 20#20: *1 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 192.168.65.1, server: localhost, request: "POST /api/auth/login HTTP/1.1", upstream: "http://172.20.0.5:9000/api/auth/login", host: "localhost:8000"
2025/08/27 18:17:34 [error] 21#21: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "POST /api/auth/login HTTP/1.1", upstream: "http://172.20.0.5:8000/api/auth/login", host: "localhost:8000"
2025/08/27 18:17:58 [error] 21#21: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "POST /api/auth/login HTTP/1.1", upstream: "http://172.20.0.5:8000/api/auth/login", host: "localhost:8000"
2025/08/27 18:18:06 [error] 21#21: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "POST /api/auth/login HTTP/1.1", upstream: "http://172.20.0.5:8000/api/auth/login", host: "localhost:8000"
2025/08/27 19:07:37 [error] 21#21: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "POST /api/auth/login HTTP/1.1", upstream: "http://172.20.0.5:8000/api/auth/login", host: "localhost:8000"
2025/08/27 19:08:10 [error] 21#21: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "POST /api/auth/login HTTP/1.1", upstream: "http://172.20.0.5:8000/api/auth/login", host: "localhost:8000"
2025/08/27 19:09:16 [error] 21#21: *15 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /api/health HTTP/1.1", upstream: "http://172.20.0.5:8000/api/health", host: "localhost:8000"
2025/08/27 19:17:26 [error] 21#21: *17 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /api/health HTTP/1.1", upstream: "http://172.20.0.5:8000/api/health", host: "localhost:8000"
2025/08/27 19:17:55 [error] 21#21: *19 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /api/health HTTP/1.1", upstream: "http://172.20.0.5:8000/api/health", host: "localhost:8000"
2025/08/27 19:18:05 [error] 21#21: *21 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /api/health HTTP/1.1", upstream: "http://172.20.0.5:8000/api/health", host: "localhost:8000"
2025/08/27 19:26:11 [error] 21#21: *27 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.20.0.7:80/", host: "localhost:8000"
2025/08/27 19:31:47 [error] 21#21: *34 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "HEAD / HTTP/1.1", upstream: "http://172.20.0.7:80/", host: "localhost:8000"
2025/08/27 19:31:58 [error] 21#21: *36 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /?ide_webview_request_time=1756312821861 HTTP/1.1", upstream: "http://172.20.0.7:80/?ide_webview_request_time=1756312821861", host: "localhost:8000"
2025/08/27 19:31:59 [error] 21#21: *36 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /@vite/client HTTP/1.1", upstream: "http://172.20.0.7:80/@vite/client", host: "localhost:8000", referrer: "http://localhost:8000/?ide_webview_request_time=1756312821861"
2025/08/27 19:38:36 [error] 21#21: *41 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "HEAD /telescope HTTP/1.1", upstream: "http://172.20.0.7:80/telescope", host: "localhost:8000"
2025/08/27 19:39:12 [notice] 26#26: signal process started
2025/08/27 19:42:10 [notice] 38#38: signal process started
2025/08/27 19:56:18 [error] 44#44: *78 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "POST /api/auth/login HTTP/1.1", upstream: "http://172.20.0.5:8000/api/v1/auth/login", host: "localhost:8000"
2025/08/27 19:57:40 [notice] 50#50: signal process started
2025/08/27 20:01:40 [error] 56#56: *88 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "POST /api/auth/login HTTP/1.1", upstream: "http://172.20.0.5:8000/api/v1/auth/login", host: "localhost:8000"
2025/08/27 22:45:03 [notice] 25#25: signal process started
2025/08/28 23:09:19 [error] 20#20: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.65.1, server: localhost, request: "GET /api/v1/sports/health HTTP/1.1", upstream: "http://172.20.0.7:8000/health", host: "localhost:8000"
